### PR TITLE
Detect single backticks at the start of the line

### PR DESCRIPTION
**Pull request by mik-laj**
Hello

I am working on the Apache Airflow project and one hook did not detect one error, so I am sending the patch.
Screenshot 2020-01-11 at 18 04 56

Best regards,
Kamil

### CODE REVIEW

**Comment by asottile** [comment]
awesome -- thanks for the patch!  can you add a quick test to https://github.com/pre-commit/pygrep-hooks/blob/master/tests/hooks_test.py demoing this change?

**Comment by mik-laj** [comment]
@asottile i added tests.

File: tests/hooks_test.py [hunk 1]
```diff
@@ -121,3 +121,127 @@ def test_python_no_log_warn_positive(s):
...
+        r" `*|]%h6O!(g,hNA|>w_kce>d7exP#K4f'%}21;A3q64&UVP,5xAo\"!KrW;Gx3qt77rn|\+ri@S?mNE/du<?4.f|c/%cX+}*A`Z",
+        r"`~rLE!9Yxn_THQr>?P/=@6\#yY`P",
+        r" `>yf2x-aRXe;GQ{x-!3Yy=6P%9o$7BM=a9v<y7!'N.h?aK;]4 6Ia\Ox, ;z}7$0@-=zd*r,_#CCNd<cBujRVB6oVhCu:cQV}*6=i`Y",
+        r"`T94c#wS6S>C-LQO71Qkas#4_l%%5I=:lGC)[R0c^qGc& 6{2X}?y]~IBpbyZY;`Z",
+        r" `)6*I?95H!$*l0p~\"3C*&]8O?@;R7Tvb>?4|AaV:A??:Um2{Lv|d5X=jFxr;8 *`-",
+        r"`GACiD1xA1r9u*&}HP_f];r5~-]}EXCUhPrpDzy`",
+        r" `JPdzQ2QqF_AQl=t]`",
+        r" `97P}j:' :MJhH,JnFCVlE\"pX cM/j6!s:=69Wf>Yc[5mb9z)W?Hb5m{Lt,)f,&x<mr'C3NBDMnr6<'|-b6`>",
+        r" `@ ~7JyWy/V9V<th:}y.#``S",
+        r" `- 3]1f{I**4Q).6}/#BrEPW.2^v]4n6oia8Xu_fOgh:q_E9Qxp}n:p T)*EId99,[7w^&AX(uQ_7~$)DX*z=t].qLh+ON>sMOt}`",
```
Comment (asottile) [inline tests/hooks_test.py]: I'm not interested in this test, please remove it -- tests should be obvious from reading what they are testing -- this one is nonsense

Comment (mik-laj) [reply tests/hooks_test.py]: Removed

**Comment by asottile** [comment]
thanks! this has been released as v1.4.4 🎉

**Comment by mik-laj** [comment]
What is the ETA for v1.4.4? I would like to update in my project.

**Comment by asottile** [comment]
negative four minutes ;)

**Pull request merged**